sunbird-cassandra-migration/cassandra-migration/src/main/resources/db/migration/cassandra/V1.89_cassandra.cql

```diff
@@ -0,0 +1 @@
+update system_settings set value ='{"fields":["firstName","lastName","profileSummary","avatar","countryCode","dob","email","gender","grade","language","location","phone","subject","userName","webPages","jobProfile","address","education","skills","badgeAssertions"],"publicFields":["firstName","lastName","profileSummary","userName"],"privateFields":["email","phone"],"csv":{"supportedColumns":{"NAME":"firstName","MOBILE PHONE":"phone","EMAIL":"email","SCHOOL ID":"orgId","USER_TYPE":"userType","ROLES":"roles","USER ID":"userId","SCHOOL EXTERNAL ID":"orgExternalId"},"outputColumns":{"userId":"USER ID","firstName":"NAME","phone":"MOBILE PHONE","email":"EMAIL","orgId":"SCHOOL ID","orgName":"SCHOOL NAME","userType":"USER_TYPE","orgExternalId":"SCHOOL EXTERNAL ID"},"outputColumnsOrder":["userId","firstName","phone","email","organisationId","orgName","userType","orgExternalId"],"mandatoryColumns":["firstName","userType","roles"]},"read":{"excludedFields":["avatar","jobProfile","address","education","webPages","skills"]},"framework":{"fields":["board","gradeLevel","medium","subject","id"]}}' where id='userProfileConfig';
```

Comment:
@manzarul please change system_settings to sunbird.system_settings.

Reply:
done